dsl/jvm/src/main/kotlin/io/mockk/InternalPlatformDsl.kt

```diff
@@ -232,3 +257,26 @@ class JvmCoroutineCall<T>(private val lambda: suspend () -> T) : CoroutineCall<T
         }
     }
 }
+
+// TODO this is copy-pasted from ValueClassSupport
+//      I will try to move that class so it's available here
```

Comment:
I'm not sure how to resolve this.

* `ValueClassSupport.kt` is in project `mockk-agent-jvm`
* `InternalPlatformDsl` is in project `mockk-dsl-jvm`

I'm looking for suggestions. Maybe it's okay just to have it copy and pasted?